### DISCUSSION TITLE
fltk-theme

### DISCUSSION THREAD

**Discussion by LotharBerger** [fltk-rs/fltk-rs#1182] [General]
This may have been asked already, many FLTK users must stick to C++ and cannot use fltk-rs, so would it be a great effort for you to offer your very good new themes for fltk-rs also for FLTK C++? There were many discussions recently, that FLTK has great functionality but poor looks with the standard themes, and therefore is not becoming more popular.

**Comment by MoAlyousef** [comment]
Hello

So the original theme code is in C++, and was done by [Remy Oukaour](https://github.com/roukaour/viz-brain-visualizer). It can be extracted as in the following demo:
https://github.com/MoAlyousef/fltk_theme_demo
